### DISCUSSION TITLE
Maintainers

### DISCUSSION THREAD

**Discussion by sbwalker** [oqtane/oqtane.translations#54] [General]
I have added @leigh-pointer as a co-maintainer of this repo as he has been publishing en-GB and nl-NL translations for every Oqtane release version in the past 3 years, and can provide guidance to translators on how to manage and maintain their translations.

**Comment by hishamco** [comment]
Hi @sbwalker, we need such permissions from the past in both Oqtane and related repos, at that time there were some active users but due to the delay in the response the PR didn't merge quickly, which reduced the number of contributions and interactivity IMHO, this remind me with Orchard Core at the beginning 

It's a good step to give some permissions, so the community can assist in accelerating the process. Regarding this repo while there's not much community in Arabic I didn't provide regular translations, but if you think it's a good step, I can go for it

**Reply by sbwalker** [reply]
I am not adding additional Admins to the Oqtane.Framework repo at this time. I am now full-time working on Oqtane so I do not believe there have been any delays in merging PRs for the past 12 months (with the exception of course for PRs which do not align with the roadmap or technical direction of the framework).

**Reply by sbwalker** [reply]
One of the main items of feedback about localization is that although Oqtane supports it fully and it works well, the process for translators to maintain resource files from release to release is not well defined. I believe this repo could provide much more guidance in this area - which would encourage translators to keep their resources updated and publish new Nuget translation packages for each release which can be discoverable and consumable through the Marketplace. @leigh-pointer has experience doing this for his translations for a number of years now which is why I added him as a co-maintainer to this repo.

**Reply by hishamco** [reply]
> I am now full-time working on Oqtane so I do not believe there have been any delays in merging PRs for the past 12 months

Yes I noticed your activity last year, it's much much better than a few years ago
